### PR TITLE
Auth UI 

### PR DESCRIPTION
**Pull request by surbhi84**
Pr consists of following changes:

- Added login page.
- Added signup page.
- Added useState and useReducer to take values from inputs.
- Signup button gets disabled if passwords don't match and if the number is invalid.

Please note consists of only ui changes, api calls will be added later.

### CODE REVIEW

File: src/pages/login/Login.tsx [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+              <FaEye
+                onClick={() => setIsPwdVisible((p) => !p)}
+                className="pwd-eye prim-text"
+              />
+            )}
+          </div>
+        </label>
+        <button
+          className="auth-btn border-prim mg-xs"
+          onClick={() => console.log(loginEmail, loginPwd)}
```
Comment (shraddha15-creator) [inline src/pages/login/Login.tsx:45]: Do we need console.log here? if not then remove it

File: src/pages/signup/Signup.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  const [isPwdVisible, setIsPwdVisible] = useState(false);
+  const [isConfirmPwdVisible, setIsConfirmPwdVisible] = useState(false);
+  const [signupDetails, signupDispatch] = useSignupReducer();
+
+  //   console.log(
+  //     signupDetails.contact.toString().length,
+  //     signupDetails.password !== signupDetails.confirmPassword,
+  //     "pswds",
+  //     signupDetails.password === "" || signupDetails.confirmPassword === ""
+  //   );
```
Comment (shraddha15-creator) [inline src/pages/signup/Signup.tsx:16]: remove commented code

**Review by shraddha15-creator — APPROVED** [review]
very clean code
Very well mannered 
Good folder structure

File: src/pages/login/login.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  position: absolute;
+  right: 0.2ex;
+  padding: 0.2rem;
+}
+
+.input-div {
+  min-width: 100%;
+}
+
+@media (max-width: 640px) {
```
Comment (SanketDhabarde) [inline src/pages/login/login.css:31]: You can use em's unit in media queries

**Review by SanketDhabarde — APPROVED** [review]
- Like the code structure
- Properly indented 🔥
- Nice work

**Comment by surbhi84** [comment]
will remove extra comments in the next pr.

**Pull request merged**